src/Client.php

```diff
@@ -43,6 +44,16 @@ class AvaTaxClientBase
      */
     protected $catchExceptions;
 
+    /**
+     * @var LoggerInterface The logger instance which is intended to be used for logging puposes      
+     */
+    protected $logger;
+
+    /**
+     * @var bool        The setting for whether the request and response body should be logged or not
+     */
+    protected $includeReqResInLogging;
```

Comment:
Could we make the variable name a bit more explicit, suggestion "logRequestAndResponseBody" as this will be passed in by the SDK caller

Reply:
Done.